packages/react-components/src/components/table-output-component.tsx

```diff
@@ -49,6 +53,10 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 }
             }
         };
+
+        this.props.unitController.onSelectionRangeChange(range => { this.handleTimeSelectionChange(range); });
```

Comment:
This handler is never removed from the unit controller when the table component is unmounted, so it continues to be called for closed event tables.